src/output-json-alert.c

```diff
@@ -217,6 +217,25 @@ void AlertJsonHeader(const Packet *p, const PacketAlert *pa, json_t *js)
     json_object_set_new(js, "alert", ajs);
 }
 
+static void AlertJsonParent(const Packet *p, json_t *js)
+{
+    json_t *parent = json_object();
+    if (parent == NULL)
+        return;
+
+    if (p->root == NULL)
+        return;
```

Comment:
It might be due to the fact that it is friday, but I don't understand what you mean. Could you explain?